app/wallet/control.js

```diff
@@ -127,6 +127,9 @@ export const purchaseTickets = (
   signTx
 ) =>
   new Promise((ok, fail) => {
+    if (stakepool.PoolAddress == "" || stakepool.PoolFees == 0) {
```

Comment:
I think would be better to have a check like this
```suggestion
    if (!stakepool.PoolAddress || !stakepool.PoolFees) {
```

To be sure it also caught undefined